index.html

```diff
@@ -224,9 +230,11 @@
     -->
   </section>
 
-  <section class="informative"> <h3>Use Cases</h3>
+  <section class="informative"> <h3>Scripting Use Cases</h3>
     <p>
-      The following scripting use cases are supported in this specification:
+      The business use cases listed in the [[WOT-USE-CASES]] document may be
+      implemented using this API. The following scripting use case scenarios are
+      supported in this specification.
```

Comment:
```suggestion
      implemented using this API. The following describes scripting use case scenarios
      supported in this specification.
```

What about having the final sentence in an active form?

Reply:
either way is fine with me

Reply:
@ashimura prefers the following title "Scripting Use Case Scenarios"